docs/spec/governance/transactions.md

```diff
@@ -183,8 +183,8 @@ vote on the proposal.
 ```go
   type TxGovVote struct {
     ProposalID           int64           //  proposalID of the proposal
-    Option               string          //  option from OptionSet chosen by the voter
-    Address      crypto.address //  Address of the validator voter wants to tie its vote to
+    Option               string          //  option chosen by the voter
+    ValidatorAddress      crypto.address //  Address of the validator voter wants to tie its vote to
```

Comment:
I'm assuming this is the validator owner address? Unfortunately I think we should probably include this detail if so get's really confusing when you're zooming in from the macro level - maybe `ValidatorOwnerAddr`

Reply:
and also should add detail in the comment

Reply:
@gamarin2 is handling the spec.

And crypto.Address is what we used to use before moving Address to the sdk.  Probably just a remnant in the spec.

Reply:
The spec here is not up-to-date. @sunnya97 spec being updated here https://github.com/cosmos/cosmos-sdk/pull/1129